ci/tests/general/general_alba.py

```diff
@@ -350,12 +352,27 @@ def filter_disks(disk_names, amount, disk_type):
             list_to_check = hdds.values()
 
         for disk_name in disk_names:
-            for disk in list_to_check:
-                if disk_name == disk['name']:
-                    filtered_disks.append(disk['name'])
-                    count += 1
-            if count == amount:
-                break
+            for node in list_to_check:
```

Comment:
We seem to be using only the values for the physical disks, which was already a dict to begin with, can't this be optimized?